RIOT-OS/RIOT
# drivers/hd44780: move params header at the right place

### Contribution description

While answering to a question on users ML about the hd44780 driver, I realized that this driver didn't have a params header file but that there's one in the tests application directory.

This PR is moving at the right place, in drivers, like it's done for others.




### Testing procedure

- Build should work at least (use one board with arduino feature):
```
$ make BOARD=arduino-zero -C tests/driver_hd44780
```
- The device should still work.





### Issues/PRs references

None
